states.js

```diff
@@ -1,3 +1,4 @@
+// TODO: git rm this file if we end up using countries.js
```

Comment:
Is this file not being used as of this PR?  Is there a discussion to be had around this?  Is the concern that maybe the new contries.js to too large or some such?

Reply:
it's really up to @gannan08 to make the call there. regions are now included as an array called children in countries.js. so yeah we probably should discuss removing it once we have a few more features that only countries.js can manage in the branch.

Reply:
@aljones15 I can't make this call because I don't know the entirety of the file you created. If the new file generated contains all 50 states and we no longer pull this file into our code, then it is redundant. If it's redundant, then we should delete it.

@mattcollier @aljones15 The countries.js may be too large as it exists now in its raw format, but I can't make the decision to continue using it or not yet. My decision can only be made based on what the gzip size of the file is after it is sent over the wire. Unfortunately, do not have that information yet.

Reply:
- [ ] remove postal prefix